CONTRIBUTING.rst

```diff
@@ -355,16 +355,65 @@ detailing the test coverage::
   ...
 
 Adding tests
-------------
+~~~~~~~~~~~~
 
 If you're **new to testing**, see existing test files for examples of things to do.
 **Don't let the tests keep you from submitting your contribution!**
 If you're not sure how to do this or are having trouble, submit your pull request
 anyway.
 We will help you create the tests and sort out any kind of problem during code review.
 
+Image comparison
+~~~~~~~~~~~~~~~~
+
+To run image comparisons::
+
+    $ PYTHONPATH=. pytest --mpl --pyargs networkx.drawing
```

Comment:
Do we need to have the `PYTHONPATH` declared explicitly here?

There are also used a lot in the testing section above.

Reply:
I would be +1 for cleaning this up, though note that in this case I just copied/pasted the section as-is to a different section in the document.

I'm happy to add the `PYTHONPATH` cleanups to this PR if it won't be a blocker!